### PR TITLE
Igor/ver 5 workouts srv

### PR DESCRIPTION
**Pull request by igormas**

### CODE REVIEW

File: dist/estimatedScore/estimatedScore.js [hunk 1]
```diff
@@ -196,11 +196,11 @@
...
 
     angular.module('znk.infra.estimatedScore').service('EstimatedScoreHelperSrv', [
-        'SubjectEnum', 'InfraConfigSrv',
-        function (SubjectEnum, InfraConfigSrv) {
+        'SubjectEnum', 'InfraConfigSrv', 'StorageSrv',
+        function (SubjectEnum, InfraConfigSrv, StorageSrv) {
             var EstimatedScoreHelperSrv = this;
 
-            var StorageSrv = InfraConfigSrv.getStorageService();
+            // var StorageSrv = InfraConfigSrv.getStorageService();
```
Comment (odeds) [inline dist/estimatedScore/estimatedScore.js]: remove comment

File: dist/estimatedScore/estimatedScore.js [hunk 2]
```diff
@@ -213,43 +213,47 @@
...
+                    return StudentStorageSrv.get(ESTIMATE_SCORE_PATH).then(function(estimatedScore){
+                        var defaultValues = {
+                            estimatedScores: {},
+                            sectionsRawScores:{},
+                            exercisesRawScores: {},
+                            processedExercises: []
+                        };
+
+                        _SetSubjectInitialVal(defaultValues.estimatedScores,[]);
+                        _SetSubjectInitialVal(defaultValues.sectionsRawScores,[]);
```
Comment (odeds) [inline dist/estimatedScore/estimatedScore.js]: maybe change to small s - capitalize usually means constructor function

File: dist/estimatedScore/estimatedScore.js (hunk 1 above)
Comment (igormas) [inline dist/estimatedScore/estimatedScore.js]: fixed

File: dist/estimatedScore/estimatedScore.js (hunk 2 above)
Comment (igormas) [inline dist/estimatedScore/estimatedScore.js]: Fixed

**Pull request merged**